src/Widgets/BannerHeadlineWidget/BannerHeadlineWidgetEditingConfig.ts

```diff
@@ -26,6 +26,12 @@ provideEditingConfig(BannerHeadlineWidget, {
       title: 'Style',
       description: 'Size and font of this headline. Default: Heading 2',
       values: [
+        { value: 'display-1', title: 'Display 1' },
+        { value: 'display-2', title: 'Display 2' },
+        { value: 'display-3', title: 'Display 3' },
+        { value: 'display-4', title: 'Display 4' },
+        { value: 'display-5', title: 'Display 5' },
+        { value: 'display-6', title: 'Display 6' },
```

Comment:
* [ ] FYI the docs name them "Display headings" and "Headings". Using this terminology may be easier to understand for an editor. Just "Display" is weird if you're not an expert.

https://getbootstrap.com/docs/5.0/content/typography/#headings
https://getbootstrap.com/docs/5.0/content/typography/#display-headings

* [ ] FYI not sure if putting display headings on top makes sense. They are now loosely sorted by size but it could make more sense to put the frequently/typically used ones further up.